docs/optimization_guide/ptq_introduction.md

```diff
@@ -0,0 +1,31 @@
+# Quantizing Models Post-training {#ptq_introduction}
+
+@sphinxdirective
+
+.. toctree::
+   :maxdepth: 1
+   :hidden:
+
+   pot_introduction
+   nncf_ptq_introduction
+
+@endsphinxdirective
+
+Post-training model optimization is the process of applying special methods that transform the model into a more hardware-friendly representation without retraining or fine-tuning. The most popular and widely-spread method here is 8-bit post-training quantization because it is:
+* Easy-to-use
+* It does not hurt accuracy a lot
+* Provides significant performance improvement
+* Suites to many HW available in stock since most of it supports 8-bit computation natively
```

Comment:
```suggestion
* It is easy-to-use.
* It does not hurt accuracy a lot.
* It provides significant performance improvement.
* It suites many hardware available in stock since most of them support 8-bit computation natively.
```